### DISCUSSION TITLE
Example of Set Variable activity in Elsa 3.1.3 designer?

### DISCUSSION THREAD

**Discussion by shxylgithub** [elsa-workflows/elsa-core#5712] [General]
Example of Set Variable activity in Elsa 3.1.3 designer?

**Comment by glime-ai** [comment]
To provide an example of the Set Variable activity in Elsa 3.1.3 designer, I would need to refer to the official documentation or any available resources that include examples of using the Set Variable activity specifically in Elsa 3.1.3. However, based on the context provided, there is no direct example or mention of the Set Variable activity for Elsa 3.1.3 in the provided sources.

In general, setting a variable in Elsa workflows typically involves defining the variable and then using an activity to set its value. This can be done programmatically or through the designer interface. For instance, in a programmatic workflow, you might define a variable and then use an activity to set its value like this:

```csharp
var workflow = new Sequence
{
    Variables = { new Variable<string>("MyVariable", "Initial Value") },
    Activities =
    {
        new SetVariable
        {
            VariableName = "MyVariable",
            ValueExpression = new JavaScriptExpression<string>("'New Value'")
        },
        // Other activities...
    }
};
```

In the designer, you would drag the Set Variable activity onto the canvas and configure its properties, such as the variable name and the value to set, using the designer's UI.

Since I do not have access to the latest documentation or examples specific to Elsa 3.1.3, I recommend checking the official Elsa documentation or the Elsa GitHub repository for the most up-to-date information on using the Set Variable activity in the designer interface.

If you need further assistance or specific details, please let me know, and I will do my best to provide the information you are looking for.

This comment was generated by [Glime](https://glimelab.ai/?utm_source=dRbvTb7IhCSYiI&utm_medium=github).
